src/Psalm/Issue/RiskyTruthyFalsyComparison.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Psalm\Issue;
+
+use Psalm\CodeLocation;
+
+final class RiskyTruthyFalsyComparison extends CodeIssue
+{
+    public const ERROR_LEVEL = 4;
```

Comment:
I think the level should be adjusted to something like 2. It may find bug but it's pretty opiniated IMHO